### DISCUSSION TITLE
Upgraded to death by version 13.1.0-DE

### DISCUSSION THREAD

**Discussion by Pewibe** [arendst/Tasmota#19322] [Support request] [answered]
I upgraded a Sonoff Basic R2 from 12.2.0-DE to 12.4.0-minimal per web interface until it came up with web interface. With no power off I choose upgrade again and selected file version 13.1.0-DE. After this second step the device doesn't come up (no response on ping / http). 
I did same procedure with another Sonoff Basic R2 with same result as before. Both devices have been upgraded per web interface for several times in the past with no problem. A power off reset doesn't help. 
Does anybody knows what to do? Who had similar Problems with the actual version?

**Comment by tsmoneta** [comment]
I don't have a Sonoff Basic R2, but I do have 102 assorted Sonoff and non-Sonoff ESP8286 and ESP32 modules. All were updated to 13.1.0 today without issue.

So it sounds like it maybe a Sonoff Basic R2 specific problem or incompatibility.

**Reply by sfromis** [reply]
Typical Sonoff Basic R2 issue is "crap device", hardly able to power itself. May have better chances sticking with older Tasmota versions, or tasmota-lite.

**Comment by barbudor** [comment]
Shouldn't be complicated to get it on serial to check what actually happens on boot, and then eventually reflash from scratch.

**Comment by Pewibe** [comment]
@tsmoneta Your suggestion "a Sonoff Basic R2 specific problem" is my believe, too because all other types of hardware went well. The processor is ESP8266EX. What I'm wondering: A Sonoff Basic RF R2 (additional RF piggy back board) is live with version 13.1.0.  
@sfromis As I wrote in my initial text, older versions until the last 12.X are running pretty well. 
@barbudor I will do so. Device LED is flashing 1 times per 1.5 sec but it's not in AP mode. Seems that my WLAN config is lost. 
I will come back after further analyses.

**Reply by sfromis** [reply]
> As I wrote in my initial text, older versions until the last 12.X are running pretty well.

Sure, that's what I meant by specifically suggesting older versions for your crap device. What I mean is that the problem is not Tasmota, but bad hardware, where older versions of Tasmota may draw a bit less power when booting.

**Comment by Pewibe** [comment, marked as answer]
1. After connecting via serial and using Tasmota web installer I could enter status command and got this result:
`
00:07:28.485 RSL: STATUS0 = {"Status":{"Module":1,"DeviceName":"Tasmota","FriendlyName":["Tasmota"],"Topic":"tasmota_8A2ED1","ButtonTopic":"0", . . . ,"StatusRetain":0},
"StatusPRM":{. . . ,"RestartReason":"Power On","Uptime":"0T00:07:29","StartupUTC":"","Sleep":50,"CfgHolder":4617,"BootCount":2,"BCResetTime":"1970-01-01T00:00:00","SaveCount":7,"SaveAddress":"F5000"},
"StatusFWR":{"Version":"13.1.0(tasmota)","BuildDateTime":"2023.08.15 08:03:05","Boot":31,"Core":"2_7_4_9","SDK":"2.2.2-dev(38a443e)","CpuFrequency":80,"Hardware":"ESP8266EX","CR":"350/699"},
"StatusMEM":{"ProgramSize":635,"Free":368,"Heap":26,"ProgramFlashSize":1024,"FlashSize":1024,"FlashChipId":"14405E","FlashFrequency":40,"FlashMode":"DOUT","Features":["00000407","8F9AC787","04368001","000000CF","010013C0","C000F981","00004004","00001000","54000020","00000080"], . . .},
"StatusNET":{"Hostname":"tasmota-8A2ED1-3793","IPAddress":"0.0.0.0","Gateway":"192.168.1.1","Subnetmask":"255.255.255.0","DNSServer1":"192.168.1.1","DNSServer2":"0.0.0.0","Mac":"EC:FA:BC:8A:2E:D1","Webserver":2,"HTTP_API":1,"WifiConfig":4,"WifiPower":17.0}
`
I can recognize that my WLAN parameters are missing, and they appear as setup for AP mode. 
2. Web installer's "connect to WLAN" found the network but was unable to connect. The log says "timeout". 
--> Problem: The entered password cannot be verified!
3. After entering my WLAN parameters per console the version 13.1.0 was running fine on Sonoff Basic. 
4. Taking away serial connection (including 3.3 V power supply) and enable 230 V power connection, the device was dead again (no WLAN connection). I tried several Tasmota-DE versions with 3.3 VDC or 230 VAC and figured out that 12.2.0 is the highest version running without problems. 
--> So it looks like a Sonoff Basic specific problem and a warning should be shown at corresponding hardware page. (@blakadder)

**Reply by sfromis** [reply]
The usual case is not that it is about all Sonoff Basic devices, but depends on aging of the bad hardware.

**Comment by ArlindoFNeto** [comment]
I'm experiencing the same problem with two Sonoff Basic devices. The highest version that works well is 12.2.0. In my case, I didn't investigate the problem like you did—I simply flashed OTA from version 11.1.0 to 13.1, and the device stopped working. Eventually, I came across your message and reverted to the previous version until 12.2.0, which made the device work again. I tried the same upgrade on another Sonoff Basic and encountered the same issue.

**Reply by sfromis** [reply]
Consider throwing out the garbage :wink:

**Reply by ArlindoFNeto** [reply]
Send me a new one, and I'll be happy to throw the current one in the garbage! :rofl:

**Comment by oywino** [comment]
It's quite clear that "something" in the Tasmota FW after 12.2.0 causes the Sonoff Basic device to fail. It does work well with the lite version of the FW, but regretfully that version is not supported by the Home Assistant Tasmota Integration.
